bower.json

```diff
@@ -1,8 +1,6 @@
 {
   "name": "ember-cli-flash",
   "dependencies": {
-    "ember": "~2.7.0",
-    "ember-cli-shims": "0.1.1",
     "ember-qunit-notifications": "0.1.0",
```

Comment:
this can also be removed once ember-cli-qunit gets updated as it is in this PR.  We can do a separate PR to remove all the bower cruft.